flasksrc/__init__.py

```diff
@@ -39,6 +39,11 @@ def create_app(test_config=None):
     api = Api(app)
     api.add_resource(HelloReclass, '/')
 
+    from .user import ToyNetUser, ToyNetUserByUsername, ToyNetUserLogin
+    api.add_resource(ToyNetUser, '/api/user')
+    api.add_resource(ToyNetUserByUsername, '/api/username')
+    api.add_resource(ToyNetUserLogin, '/api/login')
+
```

Comment:
my ignorance here- does this just make the listed URLs available for end-user access?